src/core/context/context-api.ts

```diff
@@ -191,6 +191,8 @@ export function makeContextApi(ai: ApiInput): EdgeContext {
       // Attempt to log into duress account if duress mode is enabled:
       if (ai.props.state.clientInfo.duressEnabled) {
         const duressAppId = appId + '.duress'
+        const stash =
```

Comment:
We already called `getStashByUsername` above, so it could be `stash ?= getEmptyStash(username)` on this line.

Reply:
We need to get the stash again in order to find the `duressStash` for the case where we don't have a stash until we call `loginPassword`. So, I think you're asking for just a type assertion, which is fine. I'll fixup.

Reply:
Ah, missed that. Carry on.